src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private final User corectUser = new User(1211222121L, "notShort", "notShort", 19);
+    private final User wrongLogin = new User(1211222121L, "Short", "notShort", 19);
+    private final User wrongPassword = new User(1211222121L, "notShort", "Short", 19);
+    private final User wrongAge = new User(1211222121L, "notShort", "notShort", 17);
+    private final User emptyId = new User(null, "notShort", "notShort", 19);
+    private final User emptyLogin = new User(1211222121L, null, "notShort", 19);
+    private final User emptyPassword = new User(1211222121L, "notShort", null, 19);
+    private final User emptyAge = new User(1211222121L, "notShort", "notShort", null);
+
+    @Test
+    void register_corectUser_ok() {
```

Comment:
The method name `register_corectUser_ok` does not follow the naming convention specified in the checklist. It should be `register_correctUser_ok` to reflect the correct spelling.